### DISCUSSION TITLE
Using object position attributes (e.g Sponge position)

### DISCUSSION THREAD

**Discussion by ALaws0n** [empriselab/RCareWorld#115] [Other Q&A] [answered]
Hi, 

I have seen within RCareWorld that you can get the local position and local-to-world matrix for the robot/sponge through the object's attributes. 

Is this the correct way of acquiring an accurate position for these objects within the simulation environment? Can the position information be used as part of a solution for phase 1 of the competition?

**Comment by YoruCathy** [comment, marked as answer]
Yes, feel free to use it.
